src/runtime_src/core/edge/user/shim.cpp

```diff
@@ -153,7 +153,14 @@ int ZYNQShim::mapKernelControl(const std::vector<std::pair<uint64_t, size_t>>& o
     if ((offset_it->first & (~0xFF)) != (-1UL & ~0xFF)) {
       auto it = mKernelControl.find(offset_it->first);
       if (it == mKernelControl.end()) {
-        ptr = mmap(0, offset_it->second, PROT_READ | PROT_WRITE, MAP_SHARED, mKernelFD, offset_it->first);
+        drm_zocl_info_cu info = {offset_it->first, -1};
+        int result = ioctl(mKernelFD, DRM_IOCTL_ZOCL_INFO_CU, &info);
+        if(result) {
```

Comment:
Should use space after 'if'.
"        if (result) {"

Reply:
done